Velaptor/Reactables/OpenGLInitReactable.cs

```diff
@@ -2,34 +2,14 @@
 // Copyright (c) KinsonDigital. All rights reserved.
 // </copyright>
 
-using System.Diagnostics.CodeAnalysis;
-using Velaptor.Reactables.Core;
-using Velaptor.Reactables.ReactableData;
-
 namespace Velaptor.Reactables;
 
+using Core;
+using ReactableData;
+
 /// <summary>
 /// Creates an reactable to send push notification that OpenGL is initialized.
```

Comment:
Grammar
```suggestion
/// Creates an reactable to send a push notification that OpenGL is initialized.
```

Reply:
@CalvinWilkinson I noticed that some documentation says "a push notification" and other's say "push notifications."  Should they all be singular or plural?